tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml

```diff
@@ -277,7 +277,7 @@ batch:
     # When 'AES_GCM_SELECTCHECK' is defined, AES-GCM formal verification is executed against more parameters.
     # Each dimension only checks |evp_cipher_update_len| from AES_GCM_SELECTCHECK_START to AES_GCM_SELECTCHECK_END.
     # https://github.com/awslabs/aws-lc-verification/blob/master/SAW/proof/AES/verify-AES-GCM-selectcheck-template.txt
-    - identifier: ubuntu2004_clang10x_formal_verification_aes_gcm_selectcheck_1_172
+    - identifier: ubuntu2004_clang10x_formal_verification_aes_gcm_selectcheck_1_140
```

Comment:
Almost at the point where we should just write a script that generates these stanza's.

Reply:
I thought about it, but this will go away in the near future with the unbounded proof that is faster and won't need to be split up like this.